scss/_tabs.scss

```diff
@@ -318,6 +318,30 @@ ion-tabs {
         transition: color .2s ease;
       }
     }
+   &:not(.tabs-icon-left):not(.tabs-icon-top){
+       .tab-item{
+          &.tab-item-active,
+          &.active,
+          &.activated {
+            > .tab-title, > i{
+            display:block;
+            margin-top: -$tabs-striped-border-width + 1px;
+          }
+        }
+      }
+    }
+    &.tabs-icon-left{
+       .tab-item{
+          margin-top: 1px;
+          &.tab-item-active,
+          &.active,
+          &.activated {
+            > .tab-title, > i {
+              margin-top: -0.1em;
```

Comment:
Can we put this `-0.1em` into a variable? Very specific magic number used in two places.  It could just be a variable at the top of tabs.scss.